### ISSUE TITLE
ua fix should check if livepatch has fixed the provided cve

### ISSUE THREAD

**Issue by orndorffgrant** [canonical/ubuntu-advantage-client#2181]
and provide a descriptive message if it has

**Comment by dixonmyles** [comment]
Hey there, took a stab at this. Wanted to ask a couple clarifying questions to follow up on:
- Should it be assumed that livepatch is enabled? If not enabled, should it prompt the user to enable?
- If there is a livepatch fix available, should the rest of the ua fix standard flow continue or should it simply prompt the user to install the patch?
- I noticed that the ua fix command has some active development going on to add the --dry-run flag. Should I account for this flag at all?

**Comment by orndorffgrant** [comment]
Thanks for taking a look @dixonmyles! I'll add some feedback to your PR as well, but to answer your scope-defining questions:

* Generally, let's keep this small and focus on accurate messaging. We can always expand it in the future.
* We shouldn't assume livepatch will be enabled. But we also shouldn't prompt to enable livepatch, because we don't have an easy way of answering the question: "would this CVE be fixed if we enabled livepatch?"
* If the livepatch fix is _applied_, which is what the "Fixes" list in `canonical-livepatch status` is telling us, then we should print a message that the CVE has been addressed by livepatch and end early. There's nothing else to do.
* `--dry-run` should be working properly. This piece of functionality won't actually modify the system so it doesn't need to do anything special in "dry-run" mode.
